src/Frame/Frame.cc

```diff
@@ -753,7 +746,9 @@ bool Frame::FillRegionHistogramData(
         if (!GetStokesTypeIndex(histogram_config.coordinate, stokes)) {
             continue;
         }
-        histogram_data.set_stokes(stokes);
+
+        // create and fill region histogram data message
+        CARTA::RegionHistogramData histogram_data = Message::RegionHistogramData(file_id, region_id, z, stokes, 1.0);
```

Comment:
any thoughts on whether we should use `auto` here? It should allow us to get rid of a bunch of `#includes` :shrug: @confluence @pford

Reply:
That's a good idea -- the type should be unambiguous from the wrapper name, so it shouldn't affect readability.